hop/configure.py

```diff
@@ -39,6 +39,52 @@ def _add_parser_args(parser):
         "-f", "--force", action="store_true", help="If set, overrides current configuration",
     )
 
+    setup_subparser.add_argument(
+        "-i", "--import_cred", help="Import credentilas from CSV file",
+    )
+
+
+def write_config_file(config_file, username, password):
+
+    os.makedirs(os.path.dirname(config_file), exist_ok=True)
+    with open(config_file, "w") as f:
+        toml.dump({"auth": {"username": username, "password": password}}, f)
+        logger.info(f"Generated configuration at: {config_file}")
+
+
+def configuration_setup(config_file, is_force, csv_file):
```

Comment:
What do you think about calling this `set_up_configuration` instead to match the action-noun style that the other function (`write_config_file`) has?

Reply:
okay, will do